src/speedruncompy/datatypes/defs.py

```diff
@@ -488,12 +489,41 @@ class GameBoost(Datatype):
     """Appears to always be empty"""
 
 class Region(Datatype):
-
     id: str
     name: str
     url: str
     flag: str
 
+class SocialNetwork(Datatype):
+    id: NetworkId
+    name: str
+    major: bool
+    pos: int
+    pattern: str
+
+class Area(Datatype):
+    id: str
+    name: str
+    fullName: str
+    label: str
+    flagIcon: str
+    lbFlagIcon: str
+    lbName: str
+
+class Color(Datatype):
+    id: str
+    darkColor: str
+    """Deprecated, darkColour is always used on the site"""
```

Comment:
Is this docstring meant to be under `lightColor`?

Reply:
Yes, as I copied what was elsewhere in the code